portfolio/src/main/webapp/script.js

```diff
@@ -155,4 +156,35 @@ function deleteCommentsAndUpdateDisplay() {
       alert("Error deleting comments from datastore.");
     }
   });
+}
+
+/** Get login status of the user */
+function getUserLoginStatus() {
+  fetch('/login').then(response => response.json()).then((message) => {
+    // Get login status from JSON response
+    const loginStatus = message.loginStatus;
+
+    // Create link element and set its reference to the JSON response URL 
+    const linkElement = document.createElement('a');
+    const url = message.url;
+    linkElement.href = url;
+
+    // Create paragraph element and append link to it
+    const paragraphElement = createParagraphElement("Click to ");
+    paragraphElement.appendChild(linkElement);
+
+    // Get login container
+    const loginContainer = document.getElementById('login-container');
+
+    // Add login/logout link to page based on login status
+    if (loginStatus === true) {
```

Comment:
Could this be` if (logibStatus) { .. }` or is there a truthy value even when login doesn't succeed?